app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="list-container">
+  <h1>Results for: <%= @dish%><%=" "%><%= "(#{@recipes.length})"%></h1>
+
+    <div class="row">
+      <section>
+        <%= paginate @recipes %>
+      </section>
+      <%= @recipes.each do |recipe| %>
+      <div class="col-sm">
+          <ul>
+        <li><strong><%= recipe.name %></strong></li>
+        <li><%= image_tag recipe.image_url %></li>
```

Comment:
Just a note, it's good to make the images links because it's human nature to want to click on the images.